scripts/start_rails.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b 0.0.0.0
```

Comment:
I think currently we're handling the `rm tmp/pids/server.pid` by having it in the dockerignore file, so I'm curious if something bit you here?

Reply:
Yep, I was still getting this on some up's so I included it here.